workshops/views.py

```diff
@@ -1771,6 +1771,7 @@ def workshop_staff(request):
                 people = people.filter(q, task__role__name='learner') \
                                .exclude(badges__in=instructor_badges)
 
+    emails = people.values_list('email', flat=True)
```

Comment:
Does this exclude people with empty emails or people who don't agree to be contacted (`may_contact=False`)?